integration-tests/kotlin-serialization/src/test/kotlin/io/quarkus/it/kotser/ResourceTest.kt

```diff
@@ -1,73 +1,80 @@
 package io.quarkus.it.kotser
 
 import io.quarkus.test.junit.QuarkusTest
-import io.restassured.RestAssured.given
+import io.restassured.RestAssured.*
```

Comment:
Not sure it's a change you wanted to do?

Reply:
Good catch! Fixed